client/src/components/PrivateRoute.js

```diff
@@ -1,45 +1,79 @@
-import React, { Component } from 'react'; 
-import { connect } from 'react-redux';
+import React, {
+	Component
+} from 'react';
+import {
+	connect
+} from 'react-redux';
 import PropTypes from 'prop-types';
 import {
-    Route,
-    Redirect
-  } from 'react-router-dom'
+	Route,
```

Comment:
@rahil051 why did you change the tabs of the file?

Reply:
@mashhoodr the editor reformatted the code.

Reply:
@rahil051 but you committed the code, please ensure this doesnt happen again.